options/locale/locale_en-US.ini

```diff
@@ -677,6 +677,14 @@ email_notifications.onmention = Only Email on Mention
 email_notifications.disable = Disable Email Notifications
 email_notifications.submit = Set Email Preference
 
+visibility = User visibility
+visibility.public = Public
+visibility.public_tooltip = Visible to all users
+visibility.limited = Limited
+visibility.limited_tooltip = Visible to logged in users only
+visibility.private = Private
+visibility.private_tooltip = Visible only to organization members
```

Comment:
hmm... is this true? Can you still see public repositories of private users?

Reply:
Yes, repo Visibility logic not changed.